client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -34,8 +34,8 @@ type AccentColor = {
 };
 
 const defaultAccentColor = {
-	hex: '#0675C4',
-	rgb: { r: 6, g: 117, b: 196 },
+	hex: '#113AF5',
```

Comment:
It'd be great is this component was getting the actual theme value to which it is writing to in the first place